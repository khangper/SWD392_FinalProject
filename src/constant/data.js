export const API_KEY = "API_KEY";

export const FETCH_HOME_LIVESTREAM_REQUEST = "FETCH_HOME_LIVESTREAM_REQUEST";
export const FETCH_HOME_LIVESTREAM_SUCCESS = "FETCH_HOME_LIVESTREAM_SUCCESS";
export const FETCH_HOME_LIVESTREAM_FAILURE = "FETCH_HOME_LIVESTREAM_FAILURE";

export const FETCH_HOME_FEATURED_COURSES_REQUEST =
  "FETCH_HOME_FEATURED_COURSES_REQUEST";
export const FETCH_HOME_FEATURED_COURSES_SUCCESS =
  "FETCH_HOME_FEATURED_COURSES_SUCCESS";
export const FETCH_HOME_FEATURED_COURSES_FAILURE =
  "FETCH_HOME_FEATURED_COURSES_FAILURE";
// All Live Stream
export const FETCH_STREAMS_REQUEST = "FETCH_STREAMS_REQUEST";
export const FETCH_STREAMS_SUCCESS = "FETCH_STREAMS_SUCCESS";
export const FETCH_STREAMS_FAILURE = "FETCH_STREAMS_FAILURE";

//All Instructor
export const FETCH_INSTRUCTORS_REQUEST = "FETCH_INSTRUCTORS_REQUEST";
export const FETCH_INSTRUCTORS_SUCCESS = "FETCH_INSTRUCTORS_SUCCESS";
export const FETCH_INSTRUCTORS_FAILURE = "FETCH_INSTRUCTORS_FAILURE";
export const SEARCH_INSTRUCTORS_REQUEST = "SEARCH_INSTRUCTORS_REQUEST";
export const FETCH_INSTRUCTOR_BY_ID_REQUEST = "FETCH_INSTRUCTOR_BY_ID_REQUEST";

// SaveCourse.js
export const FETCH_SAVECOURSES_REQUEST = "FETCH_SAVECOURSES_REQUEST";
export const FETCH_SAVECOURSES_SUCCESS = "FETCH_SAVECOURSES_SUCCESS";
export const FETCH_SAVECOURSES_FAILURE = "FETCH_SAVECOURSES_FAILURE";
export const DELETE_SAVECOURSE_REQUEST = "DELETE_SAVECOURSE_REQUEST";
export const DELETE_SAVECOURSE_SUCCESS = "DELETE_SAVECOURSE_SUCCESS";
export const DELETE_SAVECOURSE_FAILURE = "DELETE_SAVECOURSE_FAILURE";

// Shoppingcart.js
export const ShoppingCart_API_URL =
  "https://66908916c0a7969efd9c67ed.mockapi.io/ojt-repo/ShoppingCart";
export const ADD_TO_CART = "ADD_TO_CART";
export const REMOVE_FROM_CART = "REMOVE_FROM_CART";
export const APPLY_DISCOUNT = "APPLY_DISCOUNT";
export const ADD_TO_CART_FAILURE = " ADD_TO_CART_FAILURE";
export const ADD_TO_CART_SUCCESS = "APPLY_DISCOUNT";

//MyCourse_Course
export const FETCH_COURSES_REQUEST = "FETCH_COURSES_REQUEST";
export const FETCH_COURSES_SUCCESS = "FETCH_COURSES_SUCCESS";
export const FETCH_COURSES_FAILURE = "FETCH_COURSES_FAILURE";

export const EDIT_COURSE_REQUEST = "EDIT_COURSE_REQUEST";
export const EDIT_COURSE_SUCCESS = "EDIT_COURSE_SUCCESS";
export const EDIT_COURSE_FAILURE = "EDIT_COURSE_FAILURE";

export const DELETE_COURSE_REQUEST = "DELETE_COURSE_REQUEST";
export const DELETE_COURSE_SUCCESS = "DELETE_COURSE_SUCCESS";
export const DELETE_COURSE_FAILURE = "DELETE_COURSE_FAILURE";

//UpcommingCourse_Course
export const FETCH_UPCOMMINGCOURSES_REQUEST = "FETCH_UPCOMMINGCOURSES_REQUEST";
export const FETCH_UPCOMMINGCOURSES_SUCCESS = "FETCH_UPCOMMINGCOURSES_SUCCESS";
export const FETCH_UPCOMMINGCOURSES_FAILURE = "FETCH_UPCOMMINGCOURSES_FAILURE";

export const EDIT_UPCOMMINGCOURSE_REQUEST = "EDIT_UPCOMMINGCOURSE_REQUEST";
export const EDIT_UPCOMMINGCOURSE_SUCCESS = "EDIT_UPCOMMINGCOURSE_SUCCESS";
export const EDIT_UPCOMMINGCOURSE_FAILURE = "EDIT_UPCOMMINGCOURSE_FAILURE";

export const DELETE_UPCOMMINGCOURSE_REQUEST = "DELETE_UPCOMMINGCOURSE_REQUEST";
export const DELETE_UPCOMMINGCOURSE_SUCCESS = "DELETE_UPCOMMINGCOURSE_SUCCESS";
export const DELETE_UPCOMMINGCOURSE_FAILURE = "DELETE_UPCOMMINGCOURSE_FAILURE";
