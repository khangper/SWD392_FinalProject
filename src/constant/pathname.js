export const PATH_NAME = {
  HOME: "/home",
  EXPLORE: "/explore",
  LIVE_OUTPUT: "/live_output/:id",
  ADD_LIVE_STREAM: "/add_livestream",
  ALL_LIVE_STREAM: "/all_livestream",
  ALL_INSTRUCTOR_LIST: "/allinstructorlist/:id",
  HELP: "/help",
  REPORT_HISTORY: "/reporthistory",
  SEND_FEEDBACK: "/sendfeedback",
  OTHER_INSTRUCTOR_VIEW: "/other-instructor-view/:id",
  MY_INSTRUCTOR_PROFILE: "/my-instructor-profile",
  SAVE_COURSES: "/save-courses",
  SETTING: "/setting",
  MY_COURSE: "/courses",
  LOGIN: "/login",
  FORGOT_PASSWORD: "/forgot-password",
  SIGN_UP: "/sign-up",
  SIGN_UP_STEP: "/SignUpStep",
  THANKS: "/thanks",
  ERROR_404: "/error404",
  INVOICE: "/invoice",
  PRESS: "/press",
  SEARCH_RESULT: "/search_result",
  PAID_MEMBERSHIP: "/paidmembership",
  ABOUT_US: "/about-us",
  OUR_BLOG: "/our-blog",
  BLOG_SINGLE_VIEW: "/blog-single-view",
  CHECKOUT: "/checkout",
  SHOPPING_CART: "/shoppingcart",
  HELP_VIEW_1: "/helpview1",
  HELP_VIEW_2: "/helpview2",
  COMPANY_DETAILS: "/company-details",
  CAREER: "/career",
  APPLY_JOB: "/apply-job",
  CONTACT_US: "/contactus",
  TERMS_OF_USE: "/termsofuse",
  CERTIFICATION_FILL_FORM: "/certificationfillform",
  CERTIFICATION_TEST: "/certificates-test",
  TEST_RESULT: "/test-result",
  MESSAGE: "/messages",
  COURSES_DETAIL_VIEW: "/courses-detail-view/:id",
  PREVIEWS: "/reviews",
  COMING_SOON: "/coming-soon",
  INS_DASHBOARD: "/ins-dashboard",
  PURCHASE_COURSE: "/PurchaseCourse",
  NOTIFICATON: "/notifications",
  DOWLOADCOURSE: "/dowloadcourse",
  CERTIFICATION_CENTER: "/certificationcenter",
  SITEMAP: "/sitemap",
  EARNING: "/earning",
  PAYOUT: "/payout",
  VERTIFICATION: "/vertification",
  MYCERTIFICATIONS: "/mycertifications",
  CREDIT: "/credits",
  STATEMENTIS: "/statementis",
  STATEMENTST: "/statementst",
  ANALYTICS: "/analytics",
  CREATE_NEW_COURSE: "/create-course",
  STD_DASHBOARD: "/student-dashboard",
  NOTIFICATIONSTUDENT: "/notificationsstudent",
};
