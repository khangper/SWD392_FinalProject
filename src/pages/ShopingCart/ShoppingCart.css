.Shopping-Container {
    display: grid;
    grid-template-columns: 8fr 4fr;
    gap: 16px;
    height: calc(100% - 40px);
    margin: auto;
    padding: 0px 165px 100px 305px;
    box-sizing: border-box;
    margin-top: 90px;
}

.Shopping-conten{
    display: grid;
    grid-template-columns: 1fr;
    width: 40%;
    padding: 20px 0 0;
    background: #fff;
    margin-bottom: 17px;
    justify-items: left;
}

.Shopping-row{
    display: grid;
    grid-template-columns: 1fr;
    justify-content: center;
    margin-right: 403px;
}

img.shopping-close {
    width: 16px;
}
.Shopping-fcrse_1 {
    background: #fff;
    width: 100%;
	float: left;
    padding: 10px;
    border-radius: 3px;
	border: 1px solid #efefef;
	transition: all .2s ease-in-out;
    display: block;
}

.Shopping-fcrse_1 .shrt-cart-btn  {
    display: none;
}

.Shopping-fcrse_1:hover .Shopping-auth1lnkprce .shrt-cart-btn  {
    display: block;
	visibility: visible;
	opacity: 1;
	transition: all 0.5s ease;
}

.shrt-cart-btn:hover  {
	color: #ed2a26;
}

.Shopping-fcrse_1:hover  {
	transition: all 0.5s ease;
	 transform: translateY(-5px);
    -webkit-transform: translateY(1px);
    -moz-transform: translateY(-5px);
    -ms-transform: translateY(-5px);
    -o-transform: translateY(-5px);
}

.Shopping-hf_img {
	width: 30%;
	float: left;
	position: relative;
}

.Shopping-hf_img img {
    width: 100%;
    border-radius: 3px;
}

.Shopping-hs_content {
    padding: 0px 15px 0px 20px;
    float: left;
    width: 70%;
    position: relative;
}

.Shopping-crse14s {
	display: block;
    font-size: 16px;
    font-weight: 600;
    font-family: 'Roboto', sans-serif;
    color: #333;
    text-align: left;
    margin-bottom: 7px;
    line-height: 24px;
    
}

.Shopping-crse-cate {
    display: block;
    font-size: 13px;
    font-weight: 400;
    font-family: 'Roboto', sans-serif;
    color: #686f7a;
    text-align: left;
	padding-top: 2px;
}

.Shopping-auth1lnkprce {
    float: left;
    width: 100%;
    margin-top: 18px;
}

.Shopping-cr1fot {
    margin-bottom: 0;
    margin-top: 18px;
    font-size: 14px;
    font-family: 'Roboto', sans-serif;
    float: left;
    color: #686f7a;
    text-align: left;
    line-height: 24px;
}

.Shopping-cr1fot a {
    color: #333;
    font-weight: 500;
}

.Shopping-cr1fot a:hover {
    color: #333 !important;
	text-decoration: underline !important;
}

.Shopping-prce142 {
    float: right;
    font-size: 18px;
    font-weight: 500;
    font-family: 'Roboto', sans-serif;
    color: #333;
    text-align: right;
}

.SS-Shopping-coupon_code p {
    font-family: 'Roboto', sans-serif;
    padding: 20px 0 20px 0px;
    float: left;
    width: 100%;
}

.SS-Shopping-code-apply-btn {
    height: 40px;
    padding: 0 20px;
    border: 0;
    color: #fff;
    border-radius: 3px;
    font-family: 'Roboto', sans-serif;
    font-weight: 500;
    background: #ed2a26;
    cursor: pointer; /* Add cursor pointer for better UX */
}

.SS-Shopping-code-apply-btn:hover {
    background: #333;
}

.ui.SS-shopping.focus {
    position: relative; 
    display: inline-block; /* Ensure the container respects its content */
}

.ui.SS-left.icon.input.swdh11.swdh19 {
    display: inline-flex; 
}


.SS-srch_shopping {
    margin-bottom: 20px;
    padding: 11px 105px 11px 11px;
    width: 78%;
    background: #fff !important;
    border: 1px solid #e5e5e5 !important;
    color: #333 !important;
    box-shadow: inset 0px 1px 4px rgba(0, 0, 0, 0.02) !important;
}


.SS-input-container.flex-container {
    display: inline-flex;
}

.ui.SS-left.icon.input.swdh11.swdh19 {
    flex: 1;
}

.Shopping-code-apply-btn {
    margin-left: 10px; /* Adjust as needed for spacing */
}


.Shopping-chck-btn22{
    font-size: 14px;
    margin-top: 30px;
    font-weight: 400;
    font-family: 'Roboto', sans-serif;
    color: #fff !important;
    background: #ed2a26;
    border-radius: 3px;
    text-align: center;
    border: 0;
    width: 100%;
    float: left;
    height: 40px;
    padding: 10px 0;
}

.Shopping-chck-btn22:hover{
    background-color: #333;
}

.shopping-close-button {
    background: none;
    border: none;
    cursor: pointer;
    padding: 0;
    margin: 0;
    display: flex;
    align-items: center;
    justify-content: center;
  }
  
  .shopping-close-button img {
    width: 20px;
    height: 20px;
  }
  
  .shopping-close-button:hover img {
    opacity: 0.7;
  }