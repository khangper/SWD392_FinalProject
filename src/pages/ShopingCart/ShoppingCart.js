import React, { useState } from "react";
import "./ShoppingCart.css";
import Header from "../../components/Header-paidmember/Header";
import { Link } from "react-router-dom";
import share from "../../assets/share.png";
import saved_course from "../../assets/saved-course.png";
import not_interested from "../../assets/not-interested.png";
import report from "../../assets/report.png";
import card_icon from "..//../assets/cart-icon.png";
import Closeshopping from "..//../assets/Close-shopping.png";
import Footer from "../../components/Footer/Footer";
import { PATH_NAME } from "../../constant/pathname";

function ShoppingCart() {
  const [courses, setCourses] = useState([
    {
      id: 1,
      imageSrc:
        "https://gambolthemes.net/html-items/cursus-new-demo/images/courses/img-2.jpg",
      detailLink: "course_detail_view.html",
      title: "The Web Developer Bootcamp",
      category: "Web Development | Python",
      author: "John Doe",
      price: "$10",
    },
    // Add more course objects here
    {
      id: 2,
      imageSrc:
        "https://gambolthemes.net/html-items/cursus-new-demo/images/courses/img-3.jpg",
      detailLink: "course_detail_view.html",
      title: "The Web Developer Bootcamp",
      category: "Web Development | Python",
      author: "John Doe",
      price: "$20",
    },
  ]);

  const [couponCode, setCouponCode] = useState("");
  const [discount, setDiscount] = useState(0);

  const removeCourse = (courseId) => {
    setCourses(courses.filter((course) => course.id !== courseId));
  };

  const applyCouponCode = () => {
    const discountValue = parseFloat(couponCode);
    if (!isNaN(discountValue) && discountValue > 0) {
      setDiscount(discountValue);
    } else {
      setDiscount(0);
      alert("Invalid discount amount");
    }
  };

  // Calculate the total price
  const totalPrice = courses.reduce((acc, course) => {
    const price = parseFloat(course.price.replace("$", ""));
    return acc + price;
  }, 0);

  const discountedTotal = Math.max(totalPrice - discount, 0);

  return (
    <div className="ShoppingPage">
      <Header />

      <div className="HelpView1-grid-container">
        <div className="HelpView-title125">
          <div className="HelpView-titleleft">
            <div className="HelpView-ttl121">
              <nav aria-label="breadcrumb">
                <ol className="HelpView-breadcrumb">
                  <li className="HelpView-breadcrumb-item">
                    <a href="index.html">Home</a>
                  </li>
                  <li
                    className="HelpView-breadcrumb-item active"
                    aria-current="page"
                  >
                    /Shopping Cart
                  </li>
                </ol>
              </nav>
            </div>
          </div>
        </div>
        <div className="HelpView-title126">
          <h2>Shopping Cart</h2>
        </div>
      </div>

      {/* Body ShoppingPage */}
      <main>
        <div className="Shopping-Container">
          <div className="Billing-Details-GridContainer">
            <div className="Shopping-fcrse-container">
              {courses.map((course) => (
                <div className="Shopping-fcrse_1" key={course.id}>
                  <a href={course.detailLink} className="Shopping-hf_img">
                    <img className="cart_img" src={course.imageSrc} alt="" />
                  </a>
                  <div className="Shopping-hs_content">
                    <div className="eps_dots eps_dots10 more_dropdown">
                      <button
                        onClick={() => removeCourse(course.id)}
                        className="shopping-close-button"
                      >
                        <img src={Closeshopping} className="shopping-close" />
                      </button>
                    </div>
                    <a href={course.detailLink} className="Shopping-crse14s">
                      {course.title}
                    </a>
                    <a href="#" className="Shopping-crse-cate">
                      {course.category}
                    </a>
                    <div className="Shopping-auth1lnkprce">
                      <p className="Shopping-cr1fot">
                        By <a href="#">{course.author}</a>
                      </p>
                      <div className="Shopping-prce142">{course.price}</div>
                    </div>
                  </div>
                </div>
              ))}
            </div>
          </div>

          <div className="OrderSumury-GridContainer">
            <div className="membership_chk_bg rght1528">
              <div className="checkout_title">
                <h4>Total</h4>
                <img src="https://gambolthemes.net/html-items/cursus-new-demo/images/line.svg" />
              </div>
              <div className="order_dt_section">
                <div className="order_title">
                  <h4>Original Price</h4>
                  <div className="order_price">${totalPrice.toFixed(2)}</div>
                </div>
                <div className="order_title">
                  <h6>Discount</h6>
                  <div className="order_price">${discount.toFixed(2)}</div>
                </div>
                <div className="order_title">
                  <h2>Total</h2>
                  <div className="order_price5">
                    ${discountedTotal.toFixed(2)}
                  </div>
                </div>
                <div className="SS-Shopping-coupon_code">
                  <p>Enter a discount amount:</p>
                </div>
                <div className="SS-coupon_input">
                  <div className="SS-ui SS-shopping focus mt-15">
                    <div className="SS-input-container">
                      <input
                        className="SS-srch_shopping"
                        type="text"
                        name="couponcode"
                        id="id_coupon_code"
                        value={couponCode}
                        onChange={(e) => setCouponCode(e.target.value)}
                        required=""
                        placeholder="Enter Discount Amount"
                      />
                      <button
                        className="SS-Shopping-code-apply-btn"
                        type="button"
                        onClick={applyCouponCode}
                      >
                        Apply
                      </button>
                    </div>
                  </div>
                </div>
                <Link to={PATH_NAME.CHECKOUT} className="Shopping-chck-btn22">
                  Checkout Now
                </Link>
              </div>
            </div>
          </div>
        </div>
      </main>

      <Footer />
    </div>
  );
}

export default ShoppingCart;
