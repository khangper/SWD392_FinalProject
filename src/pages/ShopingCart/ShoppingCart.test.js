import React from "react";
import { render, screen } from "@testing-library/react";
import ShopingCart from "../ShopingCart/ShoppingCart";
import "@testing-library/jest-dom/extend-expect";
import { renderWithProviders } from "../../constant/test-util";

describe("ShopingCart", () => {
  beforeAll(() => {
    jest.spyOn(console, "error").mockImplementation(() => {});
    jest.spyOn(console, "log").mockImplementation(() => {});
  });

  afterAll(() => {
    console.error.mockRestore();
    console.log.mockRestore();
  });
  it("renders the ShopingCart component", async () => {
    const { container } = renderWithProviders(<ShopingCart />);
    expect(container).toBeDefined();
    expect(container).toMatchSnapshot();
  });
});
