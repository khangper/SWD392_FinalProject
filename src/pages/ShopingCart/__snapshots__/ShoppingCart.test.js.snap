// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShopingCart renders the ShopingCart component 1`] = `
<div>
  <div
    class="ShoppingPage"
  >
    <header
      class="header1"
    >
      <a
        href="/home"
      >
        <button
          class="header-link1"
        >
          Back To Cursus
        </button>
      </a>
      <a
        href="/home"
      >
        <img
          alt=""
          class="header-group-icon"
          src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo.svg"
        />
      </a>
      <img
        alt=""
        class="header-avatar"
        src="profile-img.jpg"
      />
    </header>
    <div
      class="HelpView1-grid-container"
    >
      <div
        class="HelpView-title125"
      >
        <div
          class="HelpView-titleleft"
        >
          <div
            class="HelpView-ttl121"
          >
            <nav
              aria-label="breadcrumb"
            >
              <ol
                class="HelpView-breadcrumb"
              >
                <li
                  class="HelpView-breadcrumb-item"
                >
                  <a
                    href="/"
                  >
                    Home
                  </a>
                </li>
                <li
                  aria-current="page"
                  class="HelpView-breadcrumb-item active"
                >
                  /Shopping Cart
                </li>
              </ol>
            </nav>
          </div>
        </div>
      </div>
      <div
        class="HelpView-title126"
      >
        <h2>
          Shopping Cart
        </h2>
      </div>
    </div>
    <main>
      <div
        class="Shopping-Container"
      >
        <div
          class="Billing-Details-GridContainer"
        >
          <div
            class="Shopping-fcrse-container"
          />
        </div>
        <div
          class="OrderSumury-GridContainer"
        >
          <div
            class="membership_chk_bg rght1528"
          >
            <div
              class="checkout_title"
            >
              <h4>
                Total
              </h4>
              <img
                alt="line"
                src="https://gambolthemes.net/html-items/cursus-new-demo/images/line.svg"
              />
            </div>
            <div
              class="order_dt_section"
            >
              <div
                class="order_title"
              >
                <h4>
                  Original Price
                </h4>
                <div
                  class="order_price"
                >
                  $
                  0.00
                </div>
              </div>
              <div
                class="order_title"
              >
                <h6>
                  Discount
                </h6>
                <div
                  class="order_price"
                >
                  $
                  0.00
                </div>
              </div>
              <div
                class="order_title"
              >
                <h2>
                  Total
                </h2>
                <div
                  class="order_price5"
                >
                  $
                  0.00
                </div>
              </div>
              <div
                class="SS-Shopping-coupon_code"
              >
                <p>
                  Enter a discount amount:
                </p>
              </div>
              <div
                class="SS-coupon_input"
              >
                <div
                  class="SS-ui SS-shopping focus mt-15"
                >
                  <div
                    class="SS-input-container"
                  >
                    <input
                      class="SS-srch_shopping"
                      id="id_coupon_code"
                      name="couponcode"
                      placeholder="Enter Discount Amount"
                      type="text"
                      value=""
                    />
                    <button
                      class="SS-Shopping-code-apply-btn"
                      type="button"
                    >
                      Apply
                    </button>
                  </div>
                </div>
              </div>
              <button
                class="Shopping-chck-btn22"
              >
                Checkout Now
              </button>
            </div>
          </div>
        </div>
      </div>
    </main>
    <footer
      class="footer footer-large-container"
    >
      <div
        class="footer-links"
      >
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/about-us"
          >
            About
          </a>
          <a
            class="link"
            href="/blog-single-view/:id"
          >
            Blog
          </a>
          <a
            class="link"
            href="/career"
          >
            Careers
          </a>
          <a
            class="link"
            href="/press"
          >
            Press
          </a>
        </div>
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/help"
          >
            Help
          </a>
          <a
            class="link"
            href="/coming-soon"
          >
            Advertise
          </a>
          <a
            class="link"
            href="/coming-soon"
          >
            Developers
          </a>
          <a
            class="link"
            href="/contactus"
          >
            Contact Us
          </a>
        </div>
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/termsofuse"
          >
            Copyright Policy
          </a>
          <a
            class="link"
            href="/termsofuse"
          >
            Terms
          </a>
          <a
            class="link"
            href="/termsofuse"
          >
            Privacy Policy
          </a>
          <a
            class="link"
            href="/sitemap"
          >
            Sitemap
          </a>
        </div>
        <div
          class="footer-column"
        >
          <button
            class="teach-button"
          >
            Teach on Cursus
          </button>
          <div
            class="language-selector"
          >
            <button>
              Language 
              <span>
                ▼
              </span>
            </button>
          </div>
        </div>
      </div>
      <div
        class="footer-bottom"
      >
        <div
          class="footer-copy"
        >
          <img
            alt=""
            src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo1.svg"
          />
          © 2024 
          <strong>
            Cursus
          </strong>
          . All Rights Reserved.
        </div>
        <div
          class="footer-social"
        >
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="facebook.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="x.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="google+.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="linkedin.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="insta.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="pinterest.png"
            />
          </a>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
